kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/wifi/WifiUtils.java

```diff
@@ -0,0 +1,24 @@
+package org.eclipse.kura.net.wifi;
+
+import org.eclipse.kura.KuraException;
+
+/**
+ * @since 2.0
+ */
+public interface WifiUtils {
```

Comment:
Based on the above comment, this service interface (an utility is generally implemented as a class with only static methods), would only manage loading the right kernel module for a mode. This is sometimes needed for WiFi interfaces where switching between access point and station mode requires loading a specific firmware. Maybe rename to `WifiDriverService`